frontend/src/components/game/Join.jsx

```diff
@@ -6,13 +6,58 @@ import ThreeScene from '../game/Game';
 
 export default function Join({ remoteGame }) {
 
-	const [gameSettings, setGameSettings] = useState(() => {
-		const fetchedSettings = localStorage.getItem('gameSettings');
-		const parsedSettings = JSON.parse(fetchedSettings);
-		console.log('fetched settings: ' + fetchedSettings);
-		return parsedSettings || "";
+	const { data: session, status } = useSession();
+	const [userId, setUserId] = useState(null);
+	const [gameSettings, setGameSettings] = useState({
+		userId: -1,
+
+		background: 0, // 0 - 3 animated, 4 - 5 static
+		palette: 0, // palette: 4 choices
+		bgColor: '#ff0000',
+		opacity: 80,
+		sparks: true,
+	
+		gameDifficulty: 4,
+		pointsToWin: 5,
+		powerUps: true
 	});
 
+	useEffect(() => {
+		if (status === "authenticated" && session) {
+			setUserId(session.user.id);
+		}
+	}, [session, status]);
+
+	useEffect(() => {
```

Comment:
What you can do here is set it to the value if you get a response, or set it to the default value that you've passed above

That way, you always have a default state in the gameSettings.